content/docs/0.13.x/operate/user_management/microsoft/index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Microsoft
+description: Setting up OpenID Connect with Microsoft
+weight: 20
+---
+
+These instructions will take you through the process of getting your Keptn authenticating with Microsoft. You will create a client within Microsoft and configure Keptn to use Microsoft for authentication.
+
+* To set up SSO via OpenID with Microsoft, you have to [register an application](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app) in order to get a client id, client secret, and a discovery endpoint.
+
+* Then the following environment variables can be set when installing Keptn:
+```
+bridge:
+  ...
+  oauth:
+    enabled: true
+    discovery: "https://login.microsoftonline.com/${directory_tenant_id}/v2.0/.well-known/openid-configuration"
```

Comment:
I would keep it consistent with the other "parameters" 
```suggestion
    discovery: "https://login.microsoftonline.com/<directory_tenant_id>/v2.0/.well-known/openid-configuration"
```

Reply:
agree. Fixed it.